src/util/simd_readme.md

```diff
@@ -0,0 +1,332 @@
+# SIMD classes for Arbor
```

Comment:
How about moving `simd_readme.md` to `src/util/simd/readme.md`, which
* moves it closer to the simd code
* makes github render it automatically when someone views the simd path.

Reply:
It's vestigial — it is wholely eclipsed by the .rst docs. Will remove!